LibOS/shim/src/shim_checkpoint.c

```diff
@@ -634,7 +634,9 @@ int init_from_checkpoint_file (const char * filename,
     struct shim_dentry * dir = NULL;
     int ret;
 
-    ret = path_lookupat(NULL, filename, LOOKUP_ACCESS|LOOKUP_DIRECTORY, &dir);
+    /* XXX: Not sure what to do here yet */
+    assert(0);
```

Comment:
It's fine to remove this assertion. This code is for migration and won't really jeopardize regular execution.

Reply:
I'd like to leave it for now.  In a few other cases, I have figured out a fix when this assertion blows.